### PR TITLE
merge back opam changes from opam-repository

### PR DESCRIPTION
**Pull request by hannesm**

### CODE REVIEW

**Comment by hannesm** [comment]
https://github.com/ocaml/opam-repository/pull/6459

File: opam [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
 maintainer: "dave.scott@citrix.com"
+authors: "Dave Scott"
+homepage: "https://github.com/mirage/xentropyd"
+bug-reports: "https://github.com/mirage/xentropyd/issues"
+license: "ISC"
 build: [
   [ "oasis" "setup" ]
   [ "ocaml" "setup.ml" "-configure" ]
   [ "ocaml" "setup.ml" "-build" ]
+  [ "cp" "main.native" "%{bin}%/xentropyd" ]
```
Comment (dsheets) [inline opam]: Even though I did the update and merged it, I note that this looks wrong and should probably use a `xentropyd.install` file to make sure removal works properly or move the `cp` into `install` and add a `remove` with `rm -f`.

Comment (hannesm) [inline opam]: my motivation here was to unbreak opam-repo and sync this opam file with opam-repo (based on a build failure of travis in shared-memory-ring) -- xentropyd is currently not used in mirage-entropy anyways -- thus once someone wants to revive it, they can happily do; there seem to be some open issues

**Comment by hannesm** [comment]
would be nice to have this merged @djs55

**Comment by djs55** [comment]
Thanks for unbreaking this. I agree the `cp main.native` is suspect -- if this is revived then this can be tweaked then.

**Pull request merged**